pkg/operator/sync.go

```diff
@@ -47,15 +47,15 @@ func (optr *Operator) syncAll(rconfig renderConfig) error {
 		if optr.inClusterBringup {
 			glog.Infof("[init mode] synced %s in %v", sf.name, time.Since(startTime))
 		}
-		optr.syncProgressingStatus()
 	}
 
 	agg := utilerrors.NewAggregate(errs)
 	if agg != nil {
 		errs = append(errs, optr.syncFailingStatus(agg))
 		agg = utilerrors.NewAggregate(errs)
 		return fmt.Errorf("error syncing: %v", agg.Error())
-	} else if optr.inClusterBringup {
+	}
+	if optr.inClusterBringup {
```

Comment:
Why this change?  The intention was to stay in "initialization" stage until we'd done a successful sync.  

(Not saying it's wrong, I just want to understand why you're making the change)

Reply:
just golang linting, the previous if branch has a return at the end, the `else` branch is therefore superfluous